admin/src/main/resources/io/buoyant/admin/js/template/router_container.handlebars

```diff
@@ -14,7 +14,11 @@
     </div>
 
     <div class="clients router-clients">
-      <div class="router-subsection-title">Clients</div>
+      <div class="pull-left router-subsection-title">Clients</div>
+      <div class="pull-right client-toggle">
+        <a target="blank" class="expand-all">expand all</a> &middot;
+        <a target="blank" class="collapse-all">collapse all</a>
+      </div>
```

Comment:
TIOLI: it might be worth hiding these two links if there are no clients, and therefore nothing to expand / collapse. Otherwise it looks like this:

![screen shot 2017-06-02 at 2 47 14 pm](https://cloud.githubusercontent.com/assets/9226/26746119/c7a59304-47a2-11e7-90c5-996300ee5e57.png)